packages/harmony/src/utils/createIcon.tsx

```diff
@@ -0,0 +1,37 @@
+import type { CSSProperties, FC } from 'react'
+
+import { IconSize, iconSizes } from '../foundations/spacing'
+
+export type IconProps = {
+  width?: number | string
+  height?: number | string
+  size?: IconSize
+  className?: string
+  style?: CSSProperties
+}
+
+export const createIcon = (src: string) => {
```

Comment:
wow look at you go! maybe something like "createImageIcon"?

Reply:
AI lol